public/intro/2_Lecture_2.md

```diff
@@ -14,9 +14,19 @@ Every expression has a *type* and a *value*. The type is fixed at compile time (
 
 ### Lvalues and Rvalues
 
-An lvalue is an expression that has an address. The value of an lvalue is defined as the value of the object at its address (try to output the expression (b = 5)). We say that a lvalue _refers_ to the object at its address.
+~~~cpp
+
+              int var = 42*320+3
+//      Lvalue ____↑        ↑______ Rvalue
+~~~
+
+Expressions can also be subdivided between __lvalues__ and __rvalues__. The names (l- and r- value) stem from the relative positions of these two kinds of expression in an assignment statement.
```

Comment:
I'd disagree on the fact that you define them by their relative position to the assignement operator, as actually I think it goes the other way around: the assignement operator _expects_ an lvalue on its left side and and rvalue on its right side. Maybe we could add instead an example with the assignment operator

Reply:
It was not intended to be a definition, just stating the fact that r and l values are on the opposite sides of the equal sign in assignment operation, because maybe not all students had realized that l stands for left and r for right